web/packages/teleport/src/Bots/Add/AddBots.tsx

```diff
@@ -32,7 +31,10 @@ export function AddBots() {
           path={cfg.getBotsNewRoute(BotFlowType.GitHubActions)}
           component={GitHubActionsFlow}
         />
-        <Route path={cfg.getBotsNewRoute()} component={AddBotsPicker} />
+        <Redirect
+          path={cfg.getBotsNewRoute()}
+          to={`${cfg.getIntegrationEnrollRoute()}?tags=bot`}
+        />
```

Comment:
with this change, the `bot` nav section on click doesn't get highlighted like in the bottom image

<img width="402" height="295" alt="image" src="https://github.com/user-attachments/assets/edf2423c-6a20-4fc7-8aad-5751f9530fcd" />

before your changes:

<img width="372" height="305" alt="image" src="https://github.com/user-attachments/assets/0fd0fc6a-1d81-4d20-abb5-81cf90fe33ec" />